pkg/api/v1/testkube/model_execution_request.go

```diff
@@ -43,3 +47,8 @@ type ExecutionRequest struct {
 	// https proxy for executor containers
 	HttpsProxy string `json:"httpsProxy,omitempty"`
 }
+
+// LocalObjectReference is used for imagePullSecrets
```

Comment:
This is autogenerated file. You need to start with OpenApi spec, then generate models, addtional methods are defined in model extended files